spec.proto

```diff
@@ -384,6 +384,20 @@ message ProtoSpecResult {
     repeated Error errors = 10;
 }
 
+/// A proto object representing the result of Scenario execution.
+message ProtoScenarioResult {
+    ProtoScenario protoScenario = 1;
+    ProtoTable scenarioDataTableRow = 2;
+    int64 scenarioDataTableRowIndex = 3;
+    ProtoTable ScenarioDataTable = 4;
+}
+
+/// A proto object representing the result of Step execution.
+message ProtoStepResult {
+    ProtoStep protoStep = 1;
```

Comment:
Does it makes sense to have ProtoItem here instead?

Reply:
I'm not sure if we need to add it as a ProtoItem. Since the result is for scenario explicitly, I don't see a point of adding as ProtoItem.